### ISSUE TITLE
missing required :container option

### ISSUE THREAD

**Issue by hgsongra** [supportify/paperclip-azure#13]
Hello @jordanyaker 

I am facing **missing required: container option** this error appears randomly on my production server.
After the certain time of period, it will start working again as it is.

I don't know what's wrong. The error says that container name is missing, though I have set the container name in my configuration as below.

_**project_root_dir/config/initializers/paperclip_azure.rb**_

```ruby
Paperclip::Attachment.default_options[:storage] = :azure
Paperclip::Attachment.default_options[:url] = ':azure_path_url'
Paperclip::Attachment.default_options[:azure_credentials] = {    
    storage_account_name: ENV['AZURE_STORAGE_ACCOUNT'],    
    access_key:           ENV['AZURE_ACCESS_KEY'],    
    container:            ENV['AZURE_CONTAINER_NAME']    
}
```
_**paperclip code in my model as below**_

```ruby
has_attached_file :avatar, styles: { medium: "250x250", thumb: "40x40>" }, 
        path: ":class/:id/:style/:basename.:extension",
        default_url: "default/:style/default-user.png"
validates_attachment_content_type :avatar, content_type: /\Aimage\/.*\z/
```
I spend lots of time to fix this but didn't find any solution yet.

Thanks.

**Comment by jordanyaker** [comment]
Hey @hgsongra! Are you still dealing with this issue?

**Comment by jordanyaker** [comment]
Fixed in 21da90f.
